### DISCUSSION TITLE
adding [shift] F3 for search function

### DISCUSSION THREAD

**Discussion by tksoh** [eteran/nedit-ng#248] [Ideas]
Given now NG runs on Windows, I would like to suggest adding F3 and shift-F3 to the find next/previous functions, since this is a common implementation on most, if not all, editors/IDE on Windows. We should preserve the original Ctrl-G and shift-Ctrl-G shortcut for backward compatibility.

Comment?

**Comment by eteran** [comment]
I'm not aware of any shortcuts which are currently using F3/Shift F3. So I see little harm in adding this additional shortcut.
